.changes/unreleased/Added-20250529-130429.yaml

```diff
@@ -0,0 +1,12 @@
+kind: Added
+body: |
+  Add support for using engine-wide default GC policy when triggering manual local cache prunes.
+
+  The manual prune API now supports an optional arg to enable honoring the engine-wide default automatic GC configuration. e.g.
```

Comment:
Should we add this to https://docs.dagger.io/configuration/cache?

Reply:
yes good idea, added!